src/Testing/Mocks/Tenant.php

```diff
@@ -29,4 +29,8 @@ class Tenant extends Model implements Contract
     protected $table = 'users';
 
     use AllowsTenantIdentification;
+
+    protected $fillable = [
```

Comment:
I'd prefer using `Tenant::unguarded()` in the createMock method of the test.

Reply:
@luceos Changed, not sure if you want it like this :)